task/buildah/0.2/buildah.yaml

```diff
@@ -309,10 +311,12 @@ spec:
         BUILDAH_ARGS+=("--skip-unused-stages=false")
       fi
 
+      VOLUME_MOUNTS="$VOLUME_MOUNTS_FROM_ENV"
```

Comment:
This will need a slight rework, Brian's PR (with changes suggested by me) has changed VOLUME_MOUNTS into an array

Reply:
Yeah, true